code/game/objects/items/defibrillator.dm

```diff
@@ -220,10 +202,6 @@
 		user.visible_message(span_warning("[icon2html(src, viewers(user))] \The [src] buzzes: Defibrillation failed: Paddles registering >100,000 ohms, Possible cause: Suit or Armor interferring."))
 		return
 
-	var/datum/internal_organ/heart/heart = H.internal_organs_by_name["heart"]
-	if(!issynth(H) && !isrobot(H) && heart && prob(25))
-		heart.take_damage(5) //Allow the defibrilator to possibly worsen heart damage. Still rare enough to just be the "clone damage" of the defib
-
```

Comment:
Not in the PR desc or changelog at all. Also don't think it's a needed change given what the pr is adding.

Reply:
ah, forgot about this one.

Yeah, I shoudl probably remove

Reply:
BM subtly buffing marines